bundles/org.smarthomej.binding.tuya/src/main/java/org/smarthomej/binding/tuya/internal/local/handlers/TuyaDecoder.java

```diff
@@ -147,7 +147,11 @@ protected void decode(@NonNullByDefault({}) ChannelHandlerContext ctx, @NonNullB
             logger.trace("{}/{}: Decoded raw payload: {}", deviceId,
                     Objects.requireNonNullElse(ctx.channel().remoteAddress(), ""), decodedMessage);
 
-            if (CommandType.STATUS.equals(commandType) || CommandType.DP_QUERY.equals(commandType)) {
+            if (CommandType.DP_QUERY.equals(commandType) && "json obj data unvalid".equals(decodedMessage)) {
```

Comment:
Is that supposed to be `unvalid` instead of `invalid`?

Reply:
Yes, you can see that in the log in the attached issue and also here: https://github.com/codetheweb/tuyapi/blob/3dec12d35e4c17f2efce0d7b0abe540d866723e1/index.js#L156

Reply:
IMO it's better to catch a `JsonSyntaxException` rather than rely on this specific error string. What if one day someone decided to fix the spelling mistake or say something completely different?

Reply:
I left this special case in place (using CONTROL is only a solution exactly this problem) and added a catch for JsonSyntaxException for all other cases and log a warning if it occurs. Since this wrong spelling is in place for years now (and has never been fixed), I believe this is the best solution.